package.json

```diff
@@ -28,6 +28,11 @@
     "react-native-windows": "^0.55.0-rc.0"
   },
   "devDependencies": {
+    "eslint": "^3.19.0",
+    "react": "^16.5.2",
```

Comment:
Why are you adding these four "react*" packages as "devDependencies"? They're already properly specified in the "peerDependencies" section. I don't think they should be duplicated here.